### ISSUE TITLE
py  t==r

### ISSUE THREAD

**Issue by Negosea** [microsoft/pyright#5765]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

If you have a question about typing or a behavior that you’re seeing in Pyright (as opposed to a bug report or enhancement request), consider posting to the [Pyright discussion forum](https://github.com/microsoft/pyright/discussions).

**Describe the bug**
A clear and concise description of the behavior you are seeing and the expected behavior along with steps to reproduce it.

**Code or Screenshots**
If possible, provide a minimal, self-contained code sample (surrounded by triple back ticks) to demonstrate the issue. The code should define or import all referenced symbols.

```python
def foo(self) -> str:
    return 3
```

If your code relies on symbols that are imported from a third-party library, include the associated import statements and specify which versions of those libraries you have installed.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension, a language server in another editor, integrated into Pylance, or the command-line tool? Which version?

**Comment by Negosea** [comment]
if 5 > 2:
   

  print("eu estou melhorando!")

if  6   < 6:
    print( 'vamos lá')#
y = 5
x = 10
print('+')
x= 5 
y =10 
z = x + y  
print("z")

**Comment by erictraut** [comment]
You haven't provided enough information for me to help you. It's not clear what problem you're experiencing or what you expect to see. Please refer to the bug template for the information we need. If you can provide more details, I'm happy to help you out. Also, please enclose your code in triple backticks so it is formatted correctly.
